File: app/src/main/java/com/main/climbingdiary/database/entities/Projekt.java
package com.main.climbingdiary.database.entities;

import org.chalup.microorm.annotations.Column;

import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class Projekt implements RouteElement{
    @Column("id") private int id;
    @Column("level") private String level;
    @Column("name") private String name;
    @Column("sektor") private String sector;
    @Column("gebiet") private String area;
    @Column("rating") private int rating;
    @Column("kommentar") private String comment;
}

File: app/src/main/java/com/main/climbingdiary/database/entities/Projekt.kt
package com.main.climbingdiary.database.entities

import org.chalup.microorm.annotations.Column

data class Projekt(
    @Column("id") var id: Int = 0,
    @Column("level") override var level: String = "8a",
    @Column("name") override var name: String? = null,
    @Column("sektor") override var sector: String? = null,
    @Column("gebiet") override var area: String? = null,
    @Column("rating") var rating: Int? = null,
    @Column("kommentar") var comment: String? = null
) : RouteElement